pymeasure/instruments/instrument.py

```diff
@@ -134,6 +134,13 @@ def id(self):
         else:
             raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
 
+    def tst(self, delay=10):
+        """ Run instrument self-test """
+        if self.SCPI:
+            return self.ask("*TST?", delay).strip()
+        else:
+            raise NotImplementedError("Non SCPI instruments require implementation in subclasses")
+        
```

Comment:
Is that a default SCPI command?
Do (almost) all SCPI compliant devices have that command?
If not, I'd put it in the particular instrument, not here.

@msmttchr , do you know more?

Reply:
According to the SCPI standard (https://www.ivifoundation.org/docs/SCPI-99.PDF), it's mandated command for IEEE 488.2 and therefore also for SCPI. I've looked through a handful of instruments I have on hand, and they all appear to support it.